k8s/crd-samples/podcstor-2.yaml

```diff
@@ -0,0 +1,73 @@
+# The example Pod utilizing the profile loaded by the sample daemon.
+
+apiVersion: v1
+kind: Pod
+metadata:
+  name: cstor
+  # Note that the Pod does not need to be in the same namespace as the loader.
+  labels:
+    app: cstor
+spec:
+#  serviceAccountName: openebs-maya-operator
+  containers:
+  - name: cstor-main-ctrl
+    image: openebs/cstor-main:ci
+    ports:
+    - containerPort: 80
+    securityContext: 
+      privileged: true
+
+    volumeMounts:
+    - name: device
+      mountPath: /host/dev     
+      mountPropagation: Bidirectional
+    - name: mount
+      mountPath: /host/mnt
+      mountPropagation: Bidirectional
+    - name: etc
+      mountPath: /host/etc
+      mountPropagation: Bidirectional
+    - name: tmp2
+      mountPath: /tmp
+      mountPropagation: Bidirectional
+
+ 
+  - name:    cstor-repl-ctrl
+    image: openebs/cstor-repl-ctrl:ci
+    ports:
+    - containerPort: 80
+    securityContext: 
+      privileged: true
+    volumeMounts:
+    - name: device
+      mountPath: /host/dev
+      mountPropagation: Bidirectional
+    - name: mount
+      mountPath: /host/mnt
+      mountPropagation: Bidirectional
+    - name: etc
+      mountPath: /host/etc
+      mountPropagation: Bidirectional
+    - name: tmp2
+      mountPath: /tmp
+      mountPropagation: Bidirectional
+
+  volumes:
+  - name: device
+    hostPath:
+      # directory location on host
+      path: /dev
+      # this field is optional
+      type: Directory
+  - name: mount
+    hostPath:
+      path: /mnt
+      type: Directory
+  - name: etc
+    hostPath:
+      path: /etc
+      type: Directory
+  - name: tmp2
+    hostPath:
+      path: /tmp2
```

Comment:
Why is this /tmp2 required

Reply:
Since /tmp on host is ephemeral, i'm creating /tmp2 on host and sharing it as /tmp into containers, since zrepl, zpool, zfs binaries creates uzfs.sock file at /tmp of container. Please confirm /tmp2 is okay or shall I go for /tmp on host?

Reply:
/tmp2 may not be ok since is very generic and can conflict with other containers on the same node. Can there be any alternative to share the sock files between side-cars?

Reply:
@kmova  Can u pls confirm, if maya-apiserver creates this deployment file or it goes into openebs-operator.yaml from user end?

Reply:
These files will be created by the maya-apiserver. A unique folder under /tmp/<pool-uid> will be provided for sharing the sock files.

Reply:
ok, i'm working on the changes